modulefiles/EMCPy.hera.modulefile

```diff
@@ -0,0 +1,5 @@
+module use /scratch1/NCEPDEV/da/Cory.R.Martin/Modulefiles/python/
```

Comment:
Rahul will say these module files don't belong in this repo, and I agree in this case, these should just be stored somewhere in instructions for the user to follow. Perhaps can add a wiki page to this repo?  Plus they will already load a previously cloned version of EMCPy.